UnicodeJsps/src/main/java/org/unicode/jsp/UnicodeSetUtilities.java

```diff
@@ -294,11 +295,15 @@ private boolean applyPropertyAlias0(
                     if (isAge) {
                         set =
                                 prop.getSet(
-                                        new UnicodePropertySymbolTable.ComparisonMatcher(
-                                                propertyValue,
+                                        new UnicodePropertySymbolTable.ComparisonMatcher<
+                                                VersionInfo>(
+                                                UnicodePropertySymbolTable.parseVersionInfoOrMax(
+                                                        propertyValue),
                                                 UnicodePropertySymbolTable.Relation.geq,
-                                                UnicodePropertySymbolTable
-                                                        .VERSION_STRING_COMPARATOR));
+                                                Comparator.nullsFirst(Comparator.naturalOrder()),
+                                                (s) ->
+                                                        UnicodePropertySymbolTable
+                                                                .parseVersionInfoOrMax(s)));
```

Comment:
Can we write this as just `UnicodePropertySymbolTable::parseVersionInfoOrMax` ?

Reply:
I tried something like that, but I had forgotten the syntax…

Someday perhaps I should learn Java.